src/content-page/content-page.gql.ts

```diff
@@ -0,0 +1,34 @@
+import { gql } from 'apollo-boost';
+
+// $userGroupIds: [Int!],
+// contentPermissionssBycontentId: { content_permission_type_id: { _in: $userGroupIds } }
+export const GET_CONTENT_PAGE_BY_PATH = gql`
+	query getContentPageByPath($path: String!) {
+		app_content(where: { path: { _eq: $path } }) {
+			title
+			content_type
+			created_at
+			depublish_at
+			description
+			id
+			is_protected
+			is_public
+			is_published
+			publish_at
+			path
+			contentBlockssBycontentId {
```

Comment:
`content_blocks: contentBlocksBycontentId`

Reply:
zou inderdaad cleaner zijn, maar dan moet ik een nieuwe typings release doen. Dus ik zal vragen dat bart het hernoemt in de database, dan kunnen we het overal clean doen